modules/components/AsyncConnect.js

```diff
@@ -56,7 +56,25 @@ export default class AsyncConnect extends Component {
   }
 
   isLoaded() {
-    return this.context.store.getState().reduxAsyncConnect.loaded;
+    const componentKeys = this.props.components.reduce((componentKeysMemo, component) => {
+      if (component && component.reduxAsyncConnect && component.reduxAsyncConnect.length) {
+        const asyncConnectComponentKeys = component.reduxAsyncConnect
+          .filter(asyncConnectComponent => !!asyncConnectComponent.key)
+          .map(asyncConnectComponent => asyncConnectComponent.key);
+
+        return [...componentKeysMemo, ...asyncConnectComponentKeys];
+      }
+
+      return componentKeysMemo;
+    }, []);
+
+    const loadState = this.context.store.getState().reduxAsyncConnect.loadState;
+    const reduxAsyncConnectComponentsLoaded = componentKeys.reduce((loadedMemo, key) => (
```

Comment:
Unfortunately I completely out of time next two week, so later I would be glad to help, since I have similar problem at the moment.